src/pages/settings/Members.tsx

```diff
@@ -0,0 +1,284 @@
+import { useRef } from 'react'
+import styled from 'styled-components'
+import { gql } from '@apollo/client'
+
+import { Typography, Skeleton, Button, InfiniteScroll } from '~/components/designSystem'
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+import ErrorImage from '~/public/images/maneki/error.svg'
+import { GenericPlaceholder } from '~/components/GenericPlaceholder'
+import { theme, NAV_HEIGHT } from '~/styles'
+import {
+  InviteItemFragmentDoc,
+  MembershipItemFragmentDoc,
+  useGetInvitesQuery,
+  useGetMembersQuery,
+} from '~/generated/graphql'
+import { InviteItem, InviteItemSkeleton } from '~/components/settings/members/InviteItem'
+import {
+  MembershipItem,
+  MembershipItemSkeleton,
+} from '~/components/settings/members/MembershipItem'
+import {
+  RevokeInviteDialog,
+  RevokeInviteDialogRef,
+} from '~/components/settings/members/RevokeInviteDialog'
+import {
+  RevokeMembershipDialog,
+  RevokeMembershipDialogRef,
+} from '~/components/settings/members/RevokeMembershipDialog'
+import {
+  CreateInviteDialog,
+  CreateInviteDialogRef,
+} from '~/components/settings/members/CreateInviteDialog'
+
+gql`
+  query getInvites($page: Int, $limit: Int) {
+    invites(page: $page, limit: $limit) {
+      metadata {
+        currentPage
+        totalPages
+        totalCount
+      }
+      collection {
+        ...InviteItem
+      }
+    }
+  }
+
+  query getMembers($page: Int, $limit: Int) {
+    memberships(page: $page, limit: $limit) {
+      metadata {
+        currentPage
+        totalPages
+      }
+      collection {
+        ...MembershipItem
+      }
+    }
+  }
+
+  ${InviteItemFragmentDoc}
+  ${MembershipItemFragmentDoc}
+`
+
+const Members = () => {
+  const { translate } = useInternationalization()
+  const revokeInviteDialogRef = useRef<RevokeInviteDialogRef>(null)
+  const revokeMembershipDialogRef = useRef<RevokeMembershipDialogRef>(null)
+  const createInviteDialogRef = useRef<CreateInviteDialogRef>(null)
+  const {
+    data: invitesData,
+    error: invitesError,
+    loading: invitesLoading,
+    refetch: invitesRefetch,
+    fetchMore: invitesFetchMore,
+  } = useGetInvitesQuery()
+  const {
+    data: membersData,
+    error: membersError,
+    loading: membersLoading,
+    refetch: membersRefetch,
+    fetchMore: membersFetchMore,
+  } = useGetMembersQuery()
+  const hasInvites = !!invitesData?.invites.metadata.totalCount
+  const invitesMetadata = invitesData?.invites.metadata
+  const membersMetadata = membersData?.memberships.metadata
+
+  return (
+    <Page>
+      <Title variant="headline">{translate('text_63208b630aaf8df6bbfb2657')}</Title>
+      <Subtitle>{translate('text_63208b630aaf8df6bbfb2659')}</Subtitle>
+      {(!!invitesLoading || !!invitesError || !!hasInvites) && (
```

Comment:
This condition is weird...
There's no case where you don't display this line no ?
Should be just 
```
<Head>
  {!!invitesLoading ? loadingBlock : normalBlock }
</Head>
```

Reply:
In case we don't have invites, we won't display this header. But if we don't have invites but are in error or loading state it should be visible